src/app/candidate-form/candidate-form.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Great how you included unit testing! Some comments would be useful to describe what the unit tests are evaluating.